db/re/item_combo_db.txt

```diff
@@ -2253,6 +2253,25 @@
 24677:24678,{ .@sum = getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L); bonus bMatkRate,2; if (.@sum>=18) { bonus bUseSPrate,40; bonus bNoGemStone; if (.@sum>=20) { bonus2 bMagicAddSize,Size_All,12; } } }
 24679:24680,{ .@sum = getequiprefinerycnt(EQI_SHADOW_ARMOR)+getequiprefinerycnt(EQI_SHADOW_SHOES); bonus2 bAddClass,Class_All,2; if (.@sum>=18) { bonus bNoKnockback; if (.@sum>=20) { bonus2 bAddEle,Ele_All,12; } } }
 24681:24682,{ .@sum = getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L); bonus2 bAddClass,Class_All,2; if (.@sum>=18) { bonus bNoKnockback; if (.@sum>=20) { bonus2 bAddEle,Ele_All,12; } } }
+24687:24688:24689:24690:24691:24692,{ bonus bAspdRate,2; bonus bVariableCastrate,10; bonus2 bIgnoreDefRaceRate,RC_All,50; bonus2 bIgnoreDefRaceRate,RC_Player_Human,-50; bonus2 bIgnoreDefRaceRate,RC_Player_Doram,-50; bonus2 bIgnoreMdefRaceRate,RC_All,50; bonus2 bIgnoreMdefRaceRate,RC_Player_Human,-50; bonus2 bIgnoreMdefRaceRate,RC_Player_Doram,-50; }
```

Comment:
```suggestion
24687:24688:24689:24690:24691:24692,{ bonus bAspd,2; bonus bVariableCastrate,-10; bonus2 bIgnoreDefRaceRate,RC_All,50; bonus2 bIgnoreDefRaceRate,RC_Player_Human,-50; bonus2 bIgnoreDefRaceRate,RC_Player_Doram,-50; bonus2 bIgnoreMdefRaceRate,RC_All,50; bonus2 bIgnoreMdefRaceRate,RC_Player_Human,-50; bonus2 bIgnoreMdefRaceRate,RC_Player_Doram,-50; }
```
For future reference I find it more practical to write the ID of the item where the combo description is put on in first

Reply:
Thanks